splink/profile_data.py

```diff
@@ -28,48 +31,86 @@ def expressions_to_sql(expressions):
     "$schema": "https://vega.github.io/schema/vega-lite/v5.9.3.json",
 }
 
-chart_path = "profile_data.json"
-_inner_chart_spec_freq = load_chart_definition(chart_path)
+_inner_chart_spec = load_chart_definition("profile_data.json")
+_distribution_plotss_plot = load_chart_definition(
+    "profile_data_distribution_plots.json"
+)
+_top_n_plot = load_chart_definition("profile_data_top_n.json")
+_bottom_n_plot = load_chart_definition("profile_data_bottom_n.json")
+_kde_plot = load_chart_definition("profile_data_kde.json")
+_correlation_plot = load_chart_definition("profile_data_correlation_heatmap.json")
```

Comment:
This isn't used in the script. The linter will fail if you try to run it.

Reply:
Code left in by mistake from larger PR- removed